### DISCUSSION TITLE
EU NÃO ESTOU CONSEGUINDO RENOVAR MEU CANVA PRO ESTUDANTE. COMO FAÇO?

### DISCUSSION THREAD

**Discussion by Caio1311** [community/community#47259] [GitHub Education] [answered]
### Select Topic Area

Product Feedback

### Body

EU NÃO ESTOU CONSEGUINDO RENOVAR MEU CANVA PRO ESTUDANTE. ALGUÉM PODE ME AJUDAR?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Caio1311  👋 ! Thanks for the great question about Global Campus Benefits.

We no longer provide the Canva benefit as part of the Student Developer Pack. You can check out all the Student Developer Pack's latest offers and partners available [here](https://education.github.com/pack?sort=az#offers). For students who have already claimed for your 12 month benefit, you will continue to have access for the remainder of your 12 month offer period and can contact [Canva's support team](https://www.canva.com/help/get-in-touch/contact-support/) with any related questions.
